gst/nnstreamer/tensor_query/meson.build

```diff
@@ -1,5 +1,8 @@
 tensor_query_sources = [
-  'tensor_query_common.c'
+  'tensor_query_common.c',
```

Comment:
plz update jni to avoid build error on android.

https://github.com/nnstreamer/nnstreamer/blob/6dcebe7cc7b68311d08998132b05fc98e53f5f68/jni/nnstreamer.mk#L46